price-feeder/oracle/provider/binance.go

```diff
@@ -152,6 +132,55 @@ func (p *BinanceProvider) GetCandlePrices(pairs ...types.CurrencyPair) (map[stri
 	return candlePrices, nil
 }
 
+// SubscribeCurrencyPais subscribe all currency pairs into ticker and candle channels.
+func (p *BinanceProvider) SubscribeCurrencyPais(cps ...types.CurrencyPair) error {
```

Comment:
Same as above : 
```suggestion
func (p *BinanceProvider) SubscribeCurrencyPairs(cps ...types.CurrencyPair) error {
```

Also, if we expect all providers to have the `SubscribeCurrencyPairs` function implemented going forward, we should add it to the interface : 
```golang
// Provider defines an interface an exchange price provider must implement.
type Provider interface {
	GetTickerPrices(...types.CurrencyPair) (map[string]TickerPrice, error)
	GetCandlePrices(...types.CurrencyPair) (map[string][]CandlePrice, error)
	SubscribeCurrencyPairs(...types.CurrencyPair) error
}
```